src/test/java/org/cactoos/scalar/AndWithIndexTest.java

```diff
@@ -59,7 +58,7 @@ void iteratesListWithIndexFromBiFunc() {
             new HasValue<>(
                 Matchers.allOf(
                     Matchers.equalTo(true),
```

Comment:
@victornoel We could use here `new IsEqual<>(true)` instead of static method `Matchers.equalTo`.

Reply:
@baudoliver7 good point, actually we can even `new IsTrue()` directly :) I will do that

Reply:
@baudoliver7 actually I simplified the code so it's a bit different than proposed